benchmark/fs/bench-glob.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const path = require('path');
+
+const benchmarkDirectory = path.resolve(__dirname, '..', '..');
+
+const configs = {
+  n: [1, 10, 100],
+  dir: ['lib', 'test/parallel', 'benchmark'],
+  pattern: ['**/*', '*.js', '**/**.js'],
+  mode: ['async', 'sync'],
+  recursive: ['true', 'false'],
```

Comment:
```suggestion
  recursive: [true, false],
```

Reply:
I didn't seem to use boolean in benchmark ([error on CI](https://github.com/nodejs/node/actions/runs/10834404918/job/30063523494)), so I fixed that in [this commit](https://github.com/nodejs/node/commit/c9764467602b803dfecc3a3a501cb79d8b534ed3)! Many Thanks!